### DISCUSSION TITLE
Always returning pending acceptance

### DISCUSSION THREAD

**Discussion by SFady** [cloudbet/docs#60] [Q&A]
Hello,

My bets are always returning pending acceptance.
Status MARKET_SUSPENDED

It was working monthes ago ... :(

**Comment by kgravenreuth** [comment]
Hi @SFady 

Could you please give us some more details or examples on which specific markets do you see this issue on? It is possible for live events to have high market suspensions due to price movements.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
